tools/lerno/README.md

```diff
@@ -0,0 +1,37 @@
+lerno
```

Comment:
does it depend on lerna? if yes, then the name is cute.

here's a suggestion for a cute name: "boo" - "build only once" (homage to "never build twice").